docs/mp/oci/01_oci.adoc

```diff
@@ -0,0 +1,69 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2021 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:javadoc-base-url-api: {javadoc-base-url}io.helidon.config/io/helidon/oci
+
+= Oracle Cloud Infrastructure Integration
+:h1Prefix: MP
+:description: Helidon OCI Integration
+:keywords: oci
+:common-deps-page-prefix-inc: ../../shared/dependencies/common_shared.adoc
+:feature-name: OCI Integration
+
+The Helidon MP OCI Integration provides easy access to Oracle Cloud Infrastructure.
```

Comment:
global comment - Helidon MP OCI integration provides
(no The before Heldion and do not capitalize integration)

Reply:
thanks, fixed)